tutorial/7 - advanced.test.ts

```diff
@@ -206,20 +208,20 @@ describe.skip('advanced', () => {
              */
             const myInverse$ = myAtom$.map(
                 // This first function is called when getting
-                n => -n,
-                // The second is called when setting, you may want to fix this one though
-                __YOUR_TURN__,
+                n => n + 1,
```

Comment:
It's not an inverse any more like this

Reply:
Good point, originally both the set and get method were the same and I thought that was a bit confusing. I will rename to plusOne$ (unless you think the original example was better)